### ISSUE TITLE
Translations don't work in datagrid toolbars

### ISSUE THREAD

**Issue by cawolf** [orocrm/platform#237]
Hi, we have an issue translating our ORO-CRM instance. Despite setting the default language to French (or German) after updating the respective translation package, the datagrid toolbar items are still shown in English.

I can reproduce this behaviour on your demo system at http://demo.orocrm.com/ as you can see on this screenshot:
![orocrm-demo](https://cloud.githubusercontent.com/assets/1932623/6571878/7fecfd9e-c70d-11e4-9398-46402cff620f.png)

I traced it down to the localized JS (e.g. http://demo.orocrm.com/js/translation/fr_FR.js ), in which items from the domain "jsmessages" are still english:

```json
...
"jsmessages:oro_datagrid.label_add_filter":"Manage filters",
"jsmessages:oro_datagrid.action.reset":"Reset",
"jsmessages:oro_datagrid.action.refresh":"Refresh"
...
```

Am I missing a crucial step or is this really a translation bug?

Thanks,
best regards,
Christian

**Comment by akuzmenko-magecore** [comment]
Hi @cawolf,
Yes, this looks like a bug. We'll make a fix in the upcoming release.
Thank you for highlighting this.

**Comment by cawolf** [comment]
Hi @akuzmenko-magecore,

I see the fix made it into the 1.7 release - that's great! :)

Thank you very much,
best regards,
Christian
